src/components/ProjectDetailPage/ProjectDetailPage.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { Grid, Card, CardTitle, CardBody, FlexItem, Flex } from '@patternfly/react-core';
+import { PageLayout } from './../PageLayout/PageLayout';
+import { useDataContainer } from '../../containers/DataContainer/useDataContainer';
+import { projectService } from '../../services/projectService';
+import { DataContainer } from '../../containers/DataContainer/DataContainer';
+import { ActionHeader } from '../ActionButton/ActionButton';
+
+const ProjectAttribute = ({ attribute, value }: { attribute: string; value: string }) => {
+  return (
+    <>
+      <span style={{ fontWeight: 'bold' }}>{attribute}</span>
+      <span style={{ ...(!value && { color: 'grey', fontStyle: 'italic' }) }}>{value ?? 'Empty'}</span>
+    </>
+  );
+};
+
+export const ProjectDetailPage = () => {
+  const { id } = useParams();
+
+  const dataContainer = useDataContainer((requestConfig: Object) => projectService.getProject(requestConfig, id!));
+
+  // TODO: Create a better solution than disabling the next line
+  useEffect(() => dataContainer.refresh({ id }), []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  return (
+    <DataContainer {...dataContainer} title="Project Details">
+      <PageLayout title={dataContainer.data?.name} description={dataContainer.data?.description}>
```

Comment:
`data?` using `?` should not be needed, `data` should be always valid, see https://github.com/project-ncl/pnc-web-ui-react/blob/main/src/containers/DataContainer/DataContainer.tsx#L46